pinot-common/src/main/java/org/apache/pinot/common/function/scalar/ObjectFunctions.java

```diff
@@ -94,28 +94,28 @@ private static Object coalesceVar(Object... objects) {
     return null;
   }
 
-  @ScalarFunction
+  @ScalarFunction(nullableParameters = true, names = {"case", "caseWhen", "case_when"})
   public static Object caseWhen(boolean c1, Object o1, Object oe) {
```

Comment:
Good catch on the `nullableParameters`. Can you please also annotate the object as `@Nullable`? Same for all functions, and we should also call it out in the PR description

Reply:
Added. Thank you!